packages/scenario/test/actions-events.test.ts

```diff
@@ -85,22 +83,9 @@ interface PredicateOptions {
  */
 describe('Actions/Events', () => {
 	describe('InstanceLoadEventsTest.java', () => {
-		/**
-		 * **PORTING NOTES**
-		 *
-		 * - I definitely think we should rephrase this one (and any in general
-		 *   where we have the full power of strings to reference particular aspects
-		 *   of spec; as such, same note applies for other applicable events).
-		 *
-		 * - ~~TIL there's a `odk-instance-load` event! It's different from
-		 *   `odk-instance-first-load`! 🤯~~
-		 *
-		 * - Above point preserved for posterity/context. The `odk-isntance-load`
-		 *   event was added to the spec in
-		 *   {@link https://github.com/getodk/xforms-spec/pull/324}.
-		 */
-		describe('[odk-instance-load] instance load event', () => {
-			it.fails('fires event on first load', async () => {
+		describe('odk-instance-load event', () => {
+			// ported from: https://github.com/getodk/javarosa/blob/2dd8e15e9f3110a86f8d7d851efc98627ae5692e/src/test/java/org/javarosa/core/model/actions/InstanceLoadEventsTest.java#L28
```

Comment:
I'm trying out this method of recording where the test was ported from instead of having cryptic punctuation characters. Let me know if this works for you and I'll keep rolling it out.